Neos.Fusion/Classes/Core/ObjectTreeParser/Token.php

```diff
@@ -89,16 +89,17 @@ public static function typeToString(int $type): string
     {
         $stringRepresentation = array_search($type, static::getConstants(), true);
 
-        if ($stringRepresentation === false) {
+        if (is_string($stringRepresentation) === false) {
```

Comment:
Why this? array_search returns false, in case it doesn't found the value.

Reply:
it returns `string|int|false` but so we have to tell phpstan that its not `int` as well. Thus the different check